code/modules/events/immovable_rod/immovable_rod.dm

```diff
@@ -182,7 +182,15 @@
 		SSexplosions.highturf += clong
 		return ..()
 
+	// If we Bump into the tram front or back, push the tram. Otherwise smash the object as usual.
 	if(isobj(clong))
+		if(istramwall(clong) && !special_target)
+			switch(dir)
+				if(NORTHEAST, EAST, SOUTHEAST, NORTHWEST, WEST, SOUTHWEST)
```

Comment:
This is very silly, and needs to be able to be configured on a per-tram basis.
Doesn't birdshot even have a North/South tram somewhere on it?